src/features/list-product/list-product.screen.tsx

```diff
@@ -1,12 +1,111 @@
-import React from 'react';
-import { Text, View } from 'react-native';
+import React, { useState } from 'react';
+import { View, Text, StyleSheet, FlatList, Image } from 'react-native';
+
+const ProductPage = () => {
+  const [products] = useState([
+    {
+      id: 1,
+      name: 'Product 1',
+      price: 'giá',
+      image: require('../../../pics/product1.png'),
+    },
+    {
+      id: 2,
+      name: 'Product 2',
+      price: 'giá',
+      image: require('../../../pics/product2.jpg'),
+    },
+    {
+      id: 3,
+      name: 'Product 3',
+      price: 'giá',
+      image: require('../../../pics/product3.jpg'),
+    },
+    {
+      id: 4,
+      name: 'Product 4',
+      price: 'giá',
+      image: require('../../../pics/product4.jpg'),
+    },
+    {
+      id: 5,
+      name: 'Product 5',
+      price: 'giá',
+      image: require('../../../pics/product5.jpg'),
+    },
+    {
+      id: 6,
+      name: 'Product 6',
+      price: 'giá',
+      image: require('../../../pics/product6.jpg'),
+    },
+    {
+      id: 7,
+      name: 'Product 7',
+      price: 'giá',
+      image: require('../../../pics/product7.jpg'),
+    },
+    {
+      id: 8,
+      name: 'Product 8',
+      price: 'giá',
+      image: require('../../../pics/product8.jpg'),
+    },
+  ]);
+
+  // const screenWidth = Dimensions.get('window').width;
+
+  const renderItem = ({ item }) => {
```

Comment:
Thêm khai báo dữ liệu

interface Product {
name: string;
price: number;
image: number | string;
}

```suggestion
  const renderItem = ({ item }: { item: Product }) => {
```